### PR TITLE
Add script for scheduling MM ping test 

### PR DESCRIPTION
**Pull request by baierjan**
Reference: https://progress.opensuse.org/issues/154624

Note: last commit is the important one; the rest is just `shfmt` like in #289

### CODE REVIEW

File: openqa-schedule-mm-ping-test [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+set -eux -o pipefail
+
+openqa_url=${openqa_url:-https://openqa.opensuse.org}
+distri=${distri:-opensuse}
+flavor=${flavor:-DVD}
+arch=${arch:-x86_64}
+version=${version:-Tumbleweed}
+test_name=${test_name:-ping_client}
+
+tmpfile=$(mktemp)
```
Comment (okurz) [inline openqa-schedule-mm-ping-test:11]: I suggest to use a cleanup jook, e.g.

```
trap 'rm $tmpfile' EXIT
```

And then without the rm-call after openqa-cli. No guarantee for first syntax though :)

File: openqa-schedule-mm-ping-test [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      HOSTNAME: client
+      PARALLEL_WITH: ping_server
+EOF
+
+openqa-cli schedule \
+    --monitor \
+    --host "$openqa_url" \
+    --param-file SCENARIO_DEFINITIONS_YAML="$tmpfile" \
+    DISTRI="$distri" VERSION="$version" FLAVOR="$flavor" ARCH="$arch" \
+    BUILD="$(date +%+4Y%m%d-%H%M)" _GROUP_ID=0 \
```
Comment (okurz) [inline openqa-schedule-mm-ping-test:56]: How about `date -Im`

**Review by okurz — CHANGES_REQUESTED** [review]

File: openqa-schedule-mm-ping-test (hunk 1 above)
Comment (baierjan) [reply openqa-schedule-mm-ping-test:11]: Yeah, that's a good idea

File: openqa-schedule-mm-ping-test (hunk 2 above)
Comment (baierjan) [reply openqa-schedule-mm-ping-test:56]: I was trying to not use `:` in the build as that is sometimes used as a delimiter. But I am ok with any sensible format, if you prefer ISO 8601 a will make that change.

Comment (okurz) [reply openqa-schedule-mm-ping-test]: Yes, I like ISO 8601 :)

**Review by okurz — APPROVED** [review]
I like the here-doc approach :)

File: openqa-schedule-mm-ping-test (hunk 2 above)
Comment (Martchus) [reply openqa-schedule-mm-ping-test]: Note that `date +%+4Y%m%d-%H%M` is *almost* an ISO 8601 timestamp. The only validation I see is the dash (the date and time need to be separated using a `T` instead). An ISO 8601 timestamp does not need to contain any colons as you can omit most separators (see https://en.wikipedia.org/wiki/ISO_8601).

**Review by Martchus — APPROVED** [review]

**Pull request merged**